install/kubernetes/cilium/templates/_helpers.tpl

```diff
@@ -6,20 +6,31 @@ Create chart name and version as used by the chart label.
 {{- end }}
 
 {{/*
-Render full image name from given values, e.g:
+Render full image uri from given values, e.g:
 ```
+optional 
+global.imageRegistry:
 image:
-  repository: quay.io/cilium/cilium
+  registry: quay.io
+  repository: cilium/cilium
   tag: v1.10.1
   useDigest: true
   digest: abcdefgh
 ```
-then `include "cilium.image" .Values.image`
-will return `quay.io/cilium/cilium:v1.10.1@abcdefgh`
+then `include "image.url" (list $ . .Values.image)`
+will return `quay.io/cilium/cilium:v1.10.1@abcdefgh` preferring the global.imageRegistry for all imageurls.
```

Comment:
I looks like other comment lines are kept a bit shorter, you might wanna move this one to the next line...
```suggestion
will return `quay.io/cilium/cilium:v1.10.1@abcdefgh`
preferring the global.imageRegistry for all `image.url`s.
```

Reply:
changed thanks!